### ISSUE TITLE
Black blank when open front end 

### ISSUE THREAD

**Issue by dev12345601** [wlox/wlox#142]
Black blank when open front end  pls advise why ?

**Comment by yokesh351** [comment]
Make sure you have pointed all sub domains as A record correctly and configured them properly in virtual hosts.

**Comment by dev12345601** [comment]
point subdomain already but it is still empty web page. why ?

**Comment by yokesh351** [comment]
Have you configured the virtual hosts and DB settings correctly?

**Comment by dev12345601** [comment]
Yes configured already. But this it is still show blank web page. 

May you share me configuration of  front end ?

**Comment by dev12345601** [comment]
This page isn’t working
xxxxx.com  is currently unable to handle this request.
HTTP ERROR 500

**Comment by yokesh351** [comment]
Is the admin up for you? have you configured the frontend base url there?

**Comment by dev12345601** [comment]
yes Admin is up. but front is not up.

Dir Root : /var/www/wlox/frontend/htdocs/
base ur: blank
Chat Base URL : blank

**Comment by yokesh351** [comment]
Why is the Base URL empty? Give your exchange url there

**Comment by dev12345601** [comment]
i have already set base URL: http://discountpayment.net/

testing exchange front link: http://discountpayment.net/

**Comment by yokesh351** [comment]
Will need more information. connect on skype - yokesh.sankar

**Comment by dev12345601** [comment]
added and text to you.
